src/transaction.rs

```diff
@@ -400,23 +404,37 @@ impl TxOut {
 
         // Parse destination scriptpubkey
         let script_pubkey = if let Some(Instruction::PushBytes(data)) = iter.next() {
-            Script::from(data.to_owned())
+            if data.is_empty() {
```

Comment:
Elements was modified to make 0 byte pushes legal @ https://github.com/ElementsProject/elements/pull/610/files#diff-f7ca24fb80ddba0f291cb66344ca6fcbR222

So this block needs to be reverted now.

Reply:
not merged yet! but looks like ACKs from relevant parties

Reply:
oh indeed.

Reply:
610 is now merged and this needs to be reverted now.

Reply:
https://github.com/ElementsProject/rust-elements/issues/20